docs/framework/wcf/migrating-from-net-remoting-to-wcf.md

```diff
@@ -485,7 +485,7 @@ public class RemotingServer : MarshalByRefObject
     </configuration>  
     ```  
   
-     For more information about using **Add Service Reference**, see [How to: Add, Update, or Remove a Service Reference](../Topic/How%20to:%20Add,%20Update,%20or%20Remove%20a%20Service%20Reference.md).  
+     For more information about using **Add Service Reference**, see [How to: Add, Update, or Remove a Service Reference](http://msdn.microsoft.com/library/cacc14bd-4455-4a44-be78-d2ac16113dd9).  
```

Comment:
Topic exists in docs: https://docs.microsoft.com/en-us/visualstudio/data-tools/how-to-add-update-or-remove-a-wcf-data-service-reference